src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java

```diff
@@ -36,7 +38,7 @@ public TimeoutCallableProcessingInterceptor timeoutInterceptor() {
 
     @Bean
     public EventPublishingController eventPublishingController() {
```

Comment:
I wonder why you decided to put `EventPublishingController` to `WebConfig`, not to `NakadiConfig`